src/helper.js

```diff
@@ -0,0 +1,4 @@
+// eslint-disable-next-line arrow-body-style
```

Comment:
Don't disable eslint

Reply:
without disabling eslint, it sees an error:
![image](https://github.com/mate-academy/react_tabs-js/assets/138882744/cd1e3059-b421-4b2b-b20b-a2e21d9a2f3c)